static/collection.js

```diff
@@ -0,0 +1,42 @@
+(async () => {
+async function create_all_games_table() {
+    const user = document.getElementById("user").value;
+    fetch('/all_games?user=' + user)
+        .then(async response => {
+            if (response.ok) {
+                const games = await response.json();
+
+                const table_element = document.getElementById("table");
+                let table = '';
+
+                for(let i=0; i < games.length; i++) {
```

Comment:
nit: JavaScript has a [`forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) method on the Array object that might be more readable to use than a vanilla `for` loop

Reply:
I read some [documentation](https://www.w3schools.com/jsref/jsref_foreach.asp) and have this working in the recent commit!